examples/bri-3/src/bri/utils/mapper.ts

```diff
@@ -0,0 +1,100 @@
+import { getType, Type, PropertyInfo, TypeKind } from 'tst-reflect';
+import { AccessModifier } from './access-modifiers';
+
+/**
+ * Generic mapper to convert object to Type at runtime
+ *
+ * @author : Manik-Jain
+ */
+export default class Mapper {
+  /**
+   * type convert from provided input to provided Type
+   * provides capabilities to define some explicit
+   * properties on the type
+   *
+   * Be careful with providing opts obeject as
```

Comment:
do we need opts at all?
ob**e**ject

Reply:
As mentioned in the JSDoc, this is for future proofing to handle scenarios when we want to set an explicit value on the mapped object. A minimalistic example can be setting subject type to external.